dockerfiles/ubuntu/Dockerfile

```diff
@@ -1,7 +1,7 @@
-ARG base_image
+ARG base_image=ubuntu:latest
```

Comment:
Could you follow https://github.com/concourse/docker-image-resource/commit/b172693efe49f5a0f007839afdee8416bbc55d03 instead of change this line? I couldn't recall the exact reason, it might be somewhere in our CI that depends on it.